system/BaseSpec.cfc

```diff
@@ -492,6 +492,27 @@ component{
 		return this;
 	}
 
+    public function getAnnotatedMethods(required string annotation, required struct metadata) {
```

Comment:
Can you please add the appropriate DocBox doc for the functions and arguments please. Also make sure the spacing is consistent with our other functions: `( required string annotation, required struct metadata )`